<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>402</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>411</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="playGameWith">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>221</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>发起游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptGame">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>331</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>接受游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditDeck1">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>221</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑牌组</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditDeck2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑牌组</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputId">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>120</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>输入账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comfirm">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
